pkg/checker/check_list.go

```diff
@@ -21,22 +21,16 @@ import (
 )
 
 const (
-	PhasePre  = "Pre"
-	PhasePost = "Post"
-	PhaseView = "view"
+	PhasePre      = "Pre"
+	PhasePost     = "Post"
+	PhasePostView = "PostView"
```

Comment:
What does post view means?

Reply:
by default post check does not display information, post view means post phase and view more info